integration/actions_test.go

```diff
@@ -715,7 +715,7 @@ func executeDeployPreviewAction(ctx context.Context, namespace string) error {
 
 	log.Printf("Deploying preview %s", namespace)
 	command := oktetoPath
-	args := []string{"preview", "deploy", namespace, "--scope", "personal", "--branch", "master", "--repository", fmt.Sprintf("%s/%s", githubUrl, pipelineRepo), "--wait"}
+	args := []string{"preview", "deploy", namespace, "--scope", "personal", "--branch", "main", "--repository", fmt.Sprintf("%s/%s", githubUrl, pipelineRepo), "--wait"}
```

Comment:
Just to mention that this is the branch of the repo, we weren't running it on master because it was failing but I think it was fixed.